app/src/main/java/de/tu_darmstadt/seemoo/nfcgate/nfc/chip/detectors/NXPDetector.java

```diff
@@ -36,6 +39,15 @@ public class NXPDetector extends BaseConfigLineDetector {
         put("0x0C","SN2X0");
     }};
 
+    // Different versions of libnfc use different keywords
+    private final static Set<String> KNOWN_NXP_NFC_CHIP_KEYWORDS;
```

Comment:
Please adapt your code to the surrounding code-style. Use a shorter name, use UpperCamelCase (all our constants should start with uppercase letters, all variables use camelCase, therefore UpperCamelCase for this one) as in `NXPChipMap` and then use static-initialization (the same type used in `NXPChipMap` and explained [here](https://stackoverflow.com/a/2041789))

Reply:
> Please adapt your code to the surrounding code-style. Use a shorter name, use UpperCamelCase (all our constants should start with uppercase letters, all variables use camelCase, therefore UpperCamelCase for this one) as in `NXPChipMap` and then use static-initialization (the same type used in `NXPChipMap` and explained [here](https://stackoverflow.com/a/2041789))

I can do this, but the static naming convention for Java is all caps and underscores. https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
> I can do this

Thanks

Reply:
> Use a shorter name

One small step for this commit, one giant leap for this PR.

Reply:
> > Use a shorter name
> 
> One small step for this commit, one giant leap for this PR.

Done, do I have to pick up the can too?